stock_valuation_fifo_lot/models/product.py

```diff
@@ -85,7 +85,7 @@ def _run_fifo(self, quantity, company):
         move_lines = correction_move_line or fifo_move._get_out_move_lines()
         for ml in move_lines:
             fifo_lot = ml.force_fifo_lot_id or ml.lot_id
-            ml_qty = fifo_move.product_uom._compute_quantity(ml.qty_done, self.uom_id)
+            ml_qty = ml.product_uom_id._compute_quantity(ml.qty_done, self.uom_id)
```

Comment:
I did this to solve the issue of below case.
- Receive 1 dozen (12 units in SVL)
- Deliver 1 dozen
With previous code, it will just deduct 1 unit from remaining qty of receiving SVL.

Reply:
Isn't this the issue mentioned about the Odoo core? If that is the case, we shouldn't be handling it in this module.

Anyway, I've changed the strategy to use the UoM of the product (not the move UoM) for all the added quantity columns in stock.move.line (for ease of balance comparison between stock valuation layers and stock move lines).

Reply:
The issue I mentioned was not decreasing `qty_done` after product is delivered.
This is for just receive and deliver with same `uom` that is not default `uom` of product.